homeassistant/components/lovelace/websocket.py

```diff
@@ -38,7 +38,7 @@ async def send_with_error_handling(hass, connection, msg):
             return
 
         if msg is not None:
-            await connection.send_big_result(msg["id"], result)
+            connection.send_big_result(msg["id"], result)
```

Comment:
I had to look at this for a bit to figure out why it is considered `big`. Ironically the entity registry is probably always "bigger".

Reply:
I wanted to try to understand why this was added in the first place, but it looks like its just because `json` is really slow.

 The largest response to these apis on my most complex install is ~42KiB .  The Entity registry is ~2312KiB.... which is why we now have #74443

Reply:
This was originally added in #23842